### PR TITLE
Workshop ee

### PR DESCRIPTION
**Pull request by craig-br**
###### Workshop default execution environments
- Workshop EEs will be pulled based on the {{ workshop_type }} variable
- All workshops that need a custom execution environment should:
  - Follow the naming convention {{ workshop_type }}_ee`.
    - As an example, the security workshop execution environment is `quay.io/acme_corp/security_ee:latest`
- Only the EE specified in the default values will be pulled for the student user
- Workshop EE and supported EEs will be pulled by controller for awx user
```
# workshop execution environments
f5_ee: "quay.io/acme_corp/f5_ee:latest"
security_ee: "quay.io/acme_corp/security_ee:latest"
rhel_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
network_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
rhel_90_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
windows_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
smart_mgmt_ee: "registry.redhat.io/ansible-automation-platform-20-early-access/ee-supported-rhel8:2.0.0"
```

###### Exeution environment vars
- Each workshop will need to define an execution environment default value in `control_node` and `populate_controller` roles
- The following defaults have been defined

###### ansible-navigator config
- Updated the image var to `{{ lookup('vars', workshop_type + '_ee') }` based on EE default naming convention
- Removed `mode: stdout` 

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- `control_node`  role
- `populate_controller` role

### CODE REVIEW

File: roles/control_node/meta/argument_spec.yml [hunk 1]
```diff
@@ -45,6 +45,10 @@ argument_specs:
         type: bool
         required: false
         default: false
+      securoty_workshop_ee:
```
Comment (anshulbehl) [inline roles/control_node/meta/argument_spec.yml:48]: spell check

File: roles/control_node/templates/ansible-navigator.yml.j2 [hunk 2]
```diff
@@ -4,10 +4,10 @@ ansible-navigator:
     inventories:
     - /home/{{username}}/lab_inventory/hosts
 
+  mode: stdout
+  
   execution-environment:
-{% if workshop_type in ['f5'] %}
-    image: quay.io/acme_corp/f5_modules:latest
-{% endif %}
+    image: quay.io/acme_corp/{{ workshop_type }}_ee
```
Comment (anshulbehl) [inline roles/control_node/templates/ansible-navigator.yml.j2:10]: Do we need a conditional here? what if a workshop type doesn't have an EE, example: for RHEL we want to use the default EE so we don't need this line

Comment (IPvSean) [reply roles/control_node/templates/ansible-navigator.yml.j2:10]: i wonder if we need to have a list somewhere.... like
```
rhel_ee: this
network_ee: that
f5_ee: another_thing
```

so the template just do 
image: "{{workshop_type}}_ee" and does a var lookup?

Comment (anshulbehl) [reply roles/control_node/templates/ansible-navigator.yml.j2:10]:
```
rhel_ee: {{ supported_ee }}
network_ee: {{ supported_ee }}
f5_ee: {{ f5_ee }}
security_ee: {{ security_ee }}
```
something like this then

**Comment by craig-br** [comment]
@IPvSean @anshulbehl ready for review

File: roles/control_node/tasks/30_controller.yml [hunk 3]
```diff
@@ -100,7 +100,7 @@
     name: "{{ item[1] }}"
   with_nested:
     - [ "awx", "{{ username }}" ]
```
Comment (anshulbehl) [inline roles/control_node/tasks/30_controller.yml:102]: When we are populating controller with the same EEs that means we are importing it in awx user, I would imaging we only need the workshop user import here.

**Comment by IPvSean** [comment]
this

**Pull request merged**